### PR TITLE
Documentation for R4 Immunization Write

### PR DESCRIPTION
**Pull request by Likitha-Seeram**
Adding Documentation for Immunization create.

### CODE REVIEW

File: content/millennium/r4/medications/immunization.md [hunk 1]
```diff
@@ -114,6 +114,58 @@ List an individual Immunization by its id:
...
 
+## Create
+
+Create a new Immunization.
+
+    POST /Immunization
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/immunization.md]: let's explicitly add patient: false

File: lib/resources/example_json/r4_examples_immunization.rb [hunk 2]
```diff
@@ -105,5 +105,62 @@ module Resources
...
     }
+
+    R4_IMMUNIZATION_CREATE ||= {
+      "resourceType": "Immunization",
+      "status": "completed",
+      "occurrenceDateTime": "2019-06-30T10:35:00-05:00",
+      "vaccineCode": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/72",
```
Comment (MaxPhilips) [inline lib/resources/example_json/r4_examples_immunization.rb]: change all instances of 2c400054 to 0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca

File: lib/resources/r4/immunization_create.yaml [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.encounter
+  description: The visit or admission or other contact between patient and health care provider the immunization was performed as part of.
+  action: create
+  example: |
+    {
+      "encounter": {
+          "reference": "Encounter/36209678"
```
Comment (MaxPhilips) [inline lib/resources/r4/immunization_create.yaml]: remove two extra spaces here and on L101

File: lib/resources/r4/immunization_create.yaml [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0443",
+                "code": "AP",
+                "display": "Administering Provider"
+              }
+            ],
+            "text": "Administering Provider"
+          },
+          "actor": {
+            "reference": "Practitioner/7750958",
+            "display": "Houde, Darrell"
```
Comment (MaxPhilips) [inline lib/resources/r4/immunization_create.yaml]: change this to a different name here and below

File: content/millennium/r4/medications/immunization.md (hunk 1 above)
Comment (Likitha-Seeram) [reply content/millennium/r4/medications/immunization.md]: Added. https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-3e96067fd99a2fee85880712d23a6719R125

File: lib/resources/example_json/r4_examples_immunization.rb (hunk 2 above)
Comment (Likitha-Seeram) [reply lib/resources/example_json/r4_examples_immunization.rb]: Updated the tenant key everywhere applicable.
https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-b4a3f148f7c74d28ca272418093bb464R116

File: lib/resources/r4/immunization_create.yaml (hunk 3 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization_create.yaml]: https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-6093e328274a82697fb90c9a404eda61L100

File: lib/resources/r4/immunization_create.yaml (hunk 4 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization_create.yaml]: The practitioner Id belongs to Jake. So I added his name. https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-6093e328274a82697fb90c9a404eda61R260

File: lib/resources/r4/immunization_create.yaml [hunk 5]
```diff
@@ -0,0 +1,318 @@
+---
```
Comment (ebaranowski) [inline lib/resources/r4/immunization_create.yaml]: Shouldnt have separate YAMLs based on the action. Please combine them

File: lib/resources/r4/immunization_create.yaml (hunk 4 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization_create.yaml]: My bad. Changed the names in all places. https://github.com/cerner/fhir.cerner.com/pull/276/commits/4596c4fb8faff62295dcbec728c1b72dc1a1f0b9

File: lib/resources/r4/immunization_create.yaml (hunk 5 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization_create.yaml]: Removed the separated yaml and merged all in to one yaml file.
https://github.com/cerner/fhir.cerner.com/pull/276/commits/075c87c022842809e7760dfd0cfc46328d9e8c1c

**Comment by MaxPhilips** [comment]
added 67200ff with spacing nits. merging

**Comment by MaxPhilips** [comment]
post-deploy validation

Screen Shot 2020-01-29 at 3 33 43 PM

**Pull request merged**